lib/index.js

```diff
@@ -1 +1,123 @@
+const P5 = require('./p5');
+const Dimensions = require('./screen_dimensions');
+const Player = require('./player');
+const Vector = require('./vector');
+const Edge = require('./edge');
+const Point = require('./point');
 
+var s = function(sketch) {
+  var height = Dimensions.height;
+  var width = Dimensions.width;
+  sketch.setup = function() {
+    sketch.frameRate(30);
+    var cnv = sketch.createCanvas(height, width);
+  };
+
+  var player = new Player(new Vector(0,0,0), new Vector(1,0,0), 5, 45, 45);
+
+  // var p1 = new Point(new Vector(100,0,0));
```

Comment:
Yeah. I'm saving those for now so that I can render render points more easily when I need them.